src/main/resources/data/minecolonies/recipes/supplycampdeployer.json

```diff
@@ -1,12 +1,12 @@
 {
   "type": "minecraft:crafting_shaped",
   "pattern": [
-    "VV",
-    "VV"
+    "V V",
+    "VVV"
   ],
   "key": {
     "V": {
-      "item": "tfc:ceramic/large_vessel"
+      "tag": "forge:chests"
     }
   },
   "result": {
```

Comment:
Could you revert this? I changed it to make it stone-age accessible on purpose

Reply:
Reverted, also I removed the supply ship recipe per our decision.